public/js/app.js

```diff
@@ -56275,8 +56275,8 @@ __webpack_require__.r(__webpack_exports__);
 /*! no static exports found */
 /***/ (function(module, exports, __webpack_require__) {
 
-__webpack_require__(/*! C:\laragon\www\api.incharge-server\resources\js\app.js */"./resources/js/app.js");
-module.exports = __webpack_require__(/*! C:\laragon\www\api.incharge-server\resources\sass\app.scss */"./resources/sass/app.scss");
+__webpack_require__(/*! C:\laragon\www\incharge-server\resources\js\app.js */"./resources/js/app.js");
+module.exports = __webpack_require__(/*! C:\laragon\www\incharge-server\resources\sass\app.scss */"./resources/sass/app.scss");
```

Comment:
This seems like an absolute import 
This file path would only exist on your laptop`C:\laragon\www\incharge-server\resources\js\app.js`